README.md

```diff
@@ -16,29 +22,36 @@ Terraform run will fail.
 ## Inputs
 
 | Name | Description | Type | Default | Required |
-|------|-------------|------|---------|:-----:|
+|------|-------------|------|---------|:--------:|
+| name | A name for the Terraform workspace | `string` | n/a | yes |
+| repository\_name | The GitHub or GitLab repository to connect the workspace to | `string` | n/a | yes |
+| repository\_owner | The GitHub organization or GitLab namespace that owns the repository | `string` | n/a | yes |
+| terraform\_organization | The Terraform Enterprise organization to create the workspace in | `string` | n/a | yes |
```

Comment:
Some of these seem out of order...

Reply:
Yes in most repositories we create the readme with this flag: --sort-inputs-by-required, and I like this its still alphabetically ordered but it puts the required vars on top. Changed the github actions in this repo to match the github actions we use for the other mcaf repos